client/src/components/App/Content/InvestigationForm/TabManagement/TabManagementStyles.ts

```diff
@@ -5,11 +5,15 @@ const useStyles = makeStyles({
         height: '70vh',
     },
     displayedTab: {
-        height: '93%',
+        height: '90%',
         overflow: 'auto'
     },
     errorIcon:{
         color: '#ffcc00'
+    },
+    icon:{
+        height: '3.5vh',
```

Comment:
You should see whether changing the `fontSize` prop of the icon is better than changing the height value.